news/_posts/releases/2020-09-01-prestashop-1-7-7-0-beta2-release.md

```diff
@@ -0,0 +1,200 @@
+- [Computing precision: set minimum computing precision value to zero](https://github.com/PrestaShop/PrestaShop/pull/20137)
+- [Auto create symfony roles on tab register](https://github.com/PrestaShop/PrestaShop/issues/19857)
+- [Add trace information to Command/Query debugger](https://github.com/PrestaShop/PrestaShop/pull/19854)
+- [Allow modules to disable routing prefix with specific parameter](https://github.com/PrestaShop/PrestaShop/pull/19782)
+- [Update decimal library](https://github.com/PrestaShop/PrestaShop/pull/19748)
+- [Update modules (contactform/ps_imageslider/ps_sharebuttons/statspersonalinfos)](https://github.com/PrestaShop/PrestaShop/pull/19556)
+- [Introduce CompilerPass to perform ControllerAwareTrait injection](https://github.com/PrestaShop/PrestaShop/pull/18811)
+- [Add tag controller.service_arguments to PrestaShop controllers](https://github.com/PrestaShop/PrestaShop/pull/18845)
+
+**Installation:**
+- [Fix session tables charset](https://github.com/PrestaShop/PrestaShop/pull/20122)
+- [Error when installing without debug mode disabled](https://github.com/PrestaShop/PrestaShop/issues/19754)
+
+**Circuit breaker:**
+The [Circuit Breaker](https://martinfowler.com/bliki/CircuitBreaker.html) design pattern provides strategies and functions to manage the unavailability of the third-party services, read more about it [here](https://build.prestashop.com/news/resilient-php-applications/)
+- [CB - Page Login - must_have.xml API call in AdminController](https://github.com/PrestaShop/PrestaShop/issues/12831)
+- [CB - Page Login - List modules call API for Recommended Modules](https://github.com/PrestaShop/PrestaShop/issues/12833)
+- [CB - Page Login - call native.xml in Admin controller](https://github.com/PrestaShop/PrestaShop/issues/12825)
+- [CB - Page Login - call native_all.xml in AdminController](https://github.com/PrestaShop/PrestaShop/issues/12826) 
+
+## Deprecations
+- [Deprecate hookcount and add missing hook in hook.xml](https://github.com/PrestaShop/PrestaShop/pull/19657)
+- [Order hooks deprecated](https://github.com/PrestaShop/PrestaShop/pull/20311)
+- [Product::supplier_reference deprecated](https://github.com/PrestaShop/PrestaShop/pull/20198)
+- [Addons related calls have no more effect](https://github.com/PrestaShop/PrestaShop/pull/19461)
+
+## BC breaks
+- [Increase precision for DECIMAL fields in database](https://github.com/PrestaShop/PrestaShop/pull/18532)
+- [Calculator::calculateFees argument ignored](https://github.com/PrestaShop/PrestaShop/pull/20482)
+- [ModuleTabRegister constructor has a new parameter](https://github.com/PrestaShop/PrestaShop/pull/19903)
+- [Old order page files removed](https://github.com/PrestaShop/PrestaShop/pull/18984)
+
+## Changelog
+
+130+ pull requests have been merged since the first beta release and 100+ issues have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. 
```

Comment:
```suggestion
130+ pull requests have been merged since the first beta release and 100+ issues have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.7.0-beta.2) for details. 
```